src/portal/src/app/project/p2p-provider/p2p-provider.service.ts

```diff
@@ -0,0 +1,103 @@
+// Copyright Project Harbor Authors
```

Comment:
The content has only some declared methods or const, and no service is used, so there is no need to create a service.

Reply:
There same public methords